dbdev/models.py

```diff
@@ -0,0 +1,45 @@
+import psycopg2
```

Comment:
Hey, have you checked the importance of having a docstring comments on your classes, functions and also the modules?